### ISSUE TITLE
Incorrect "No unwrapped JSX text" lint error

### ISSUE THREAD

**Issue by bernardobelchior** [sayari-analytics/eslint-plugin-sayari#1]
Hey, there! 

First of all, thank very much for this lint. I'm getting the error related to React's problems with tools like Google Translate and this looks like a way forward to avoid it 😄 

I just started integrating this lint into our codebase, but I'm getting (what I think is) a false positive. I've managed to reproduce a minimal example here:

```ts
import React from 'react';

const Element = () => {
  const flag = true;

  return (
    <div>
        {flag ? (
         <>
          <div />
          <div />
         </>
      ) : null}
      <div />
    </div>
  );
};

```

The error I'm getting is `ESLint: No unwrapped JSX text(@sayari/no-unwrapped-jsx-text)`. The suggested fix is wrapping the `{ flag ... }` conditional in a `span`.

From my understading, this lint should be working on text, but it's interpreting React fragments as text. Unfortunately, I'm not familiar with how ESLint rules work, so I don't think I can contribute, but I'm imagining that now that this rule is discoverable through the React issue, more people will be facing this problem. 

If you know where the problem might be, let me know and I can try to fix and test it. 

All in all, thank you very much for this rule, it's awesome! 💯
